lib/src/ui/map_control_buttons.dart

```diff
@@ -31,15 +32,30 @@ class MapControlButtons extends StatelessWidget {
   /// This button is currently not available on web.
   final bool showTrackLocation;
 
+  /// The explanation to show when requesting location permissions.
+  final String requestPermissionsExplanation;
+
+  @override
+  State<MapControlButtons> createState() => _MapControlButtonsState();
+}
+
+class _MapControlButtonsState extends State<MapControlButtons> {
+  late final PermissionManager? permissionManager;
```

Comment:
PermissionManager needs to be nullable because it can't be initialized on web.
This allows the widget to be used across different platforms without causing errors.